src/backend/expungeservice/models/statute.py

```diff
@@ -0,0 +1,91 @@
+
+
+
+# I am fairly certain that section and subsection are irrelevant to the analyzers logic
+
+
+# the way i currently have this configured completely discards section and subsection,  cameron - jan 3 2019
+
+#todo: find out if section and subsection are relevant
+
+class Statute(object):
+    """ Statute corresponding to a law
+
+    Statutes are represented by numbers in hierarchical manner:
+    chapter.subchapter(section)(subsection) e.g. 653.412(5)(c)
+
+    Attributes:
+        chapter: An integer that specifies statute chapter.
+        subchapter: An integer that specifies statute sub-chapter.
+        section: An integer that specifies the section within sub-chapter.
+        subsection: A string of length 1 that specifies the sub-section within
+                    section.
+    """
+    def __init__(self, statute_string, chapter=None, subchapter=None, section=None, subsection=None):
+
+        self.statute_string = str(statute_string)
+        self.chapter = chapter
+        self.subchapter = subchapter
+        self.section = section
+        self.subsection = subsection
+
+        self.parse_string()
+
+        # TODO we may need to add components beyond subsection
+
+    def parse_string(self):
+        statute_string = self.statute_string
+        if len(str(statute_string))>=6: #todo: this is wrong but will kinda work for everything on our list except marijuana crimes.
+
+            statute_string = statute_string.lower() #convert to lowercase
+            statute_string = [char for char in statute_string if char not in "abcdefghijklmnopqrstuvwxyz!@#$%^&*() .,;'[]<>?:{}\""] #remove all other chars #todo: fix this it could include weird chars
+            statute_string = ''.join(statute_string)
+            statute_string = statute_string[0:6] #trim to only first 6
+
+            self.chapter = statute_string[0:3]
+            self.subchapter = statute_string[3:7]
```

Comment:
We can move the parsing method to constructor. Also we should store the chapter, subchapter as ints. Integer comparison should be faster.